starport-blog/01-index.md

```diff
@@ -0,0 +1,350 @@
+# Create posts
+
+By following this beginner tutorial, you will end up with a simple blog app that is powered by the Cosmos SDK.
+
+## Prerequisites
+
+- A [recent version of Go](https://golang.org/doc/install) installed.
+
+## Getting Started
+
+Let's get started! The first step is to install the `starport` CLI tool:
+
+### NPM
+
+```
+npm install -g @tendermint/starport
```

Comment:
is this npm or brew or both?

Reply:
It's either. Yeah, I think link to a readme makes sense.